secret_store/src/key_server_cluster/connection_trigger.rs

```diff
@@ -21,8 +21,10 @@ use std::sync::Arc;
 use ethereum_types::H256;
 use ethkey::Public;
 use key_server_cluster::{KeyServerSet, KeyServerSetSnapshot};
-use key_server_cluster::cluster::{ClusterClient, ClusterConnectionsData};
+use key_server_cluster::cluster::{ClusterConfiguration, ServersSetChangeParams};
 use key_server_cluster::cluster_sessions::AdminSession;
+use key_server_cluster::cluster_connections::{Connection};
+use key_server_cluster::cluster_connections_net::{NetConnectionsContainer};
 use types::{Error, NodeId};
 use {NodeKeyPair};
```

Comment:
unnecessary blocks for single import
```suggestion
use NodeKeyPair;
```